### PR TITLE
Add `no-wrap` option for very long strings

### PR DESCRIPTION
**Pull request by rstefan1**
https://github.com/laget-se/react-gettext-parser/issues/64

### CODE REVIEW

**Comment by rstefan1** [comment]
@alexanderwallin, could you take a look when you have time?

File: src/cli.js [hunk 1]
```diff
@@ -33,6 +33,10 @@ const args = yargs
     type: 'boolean',
     description: 'Disables line numbers in POT reference comments',
   }).argv
+  .option('no-wrap', {
+    type: 'boolean',
+    description: 'Do not break long strings into several lines',
```
Comment (alexanderwallin) [inline src/cli.js]: "Do" -> "Does" for consistency

File: src/json2pot.js [hunk 2]
```diff
@@ -61,7 +61,12 @@ export const toPot = (blocks, opts = {}) => {
     headers: transformHeaders(potJson.headers),
   }
 
-  const pot = po.compile(transformedPotJson)
+  var compilerOpts = {};
+  if (opts.noWrap == true){
```
Comment (alexanderwallin) [inline src/json2pot.js]: Strict comparison with `===`

File: src/json2pot.js [hunk 3]
```diff
@@ -61,7 +61,12 @@ export const toPot = (blocks, opts = {}) => {
     headers: transformHeaders(potJson.headers),
   }
 
-  const pot = po.compile(transformedPotJson)
+  var compilerOpts = {};
```
Comment (alexanderwallin) [inline src/json2pot.js]: `const` instead of `var`

**Comment by rstefan1** [comment]
@alexanderwallin, done

**Comment by rstefan1** [comment]
@alexanderwallin  ping

**Comment by alexanderwallin** [comment]
Sorry about the wait. v1.12.0 is published to npm.

**Pull request merged**